<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlPanel</class>
 <widget class="QMainWindow" name="ControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlPanel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGraphicsView" name="YawView">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QProgressBar" name="RollBar">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>90</y>
        <width>61</width>
        <height>281</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-90</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::BottomToTop</enum>
      </property>
      <property name="format">
       <string>m</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="DepthBar">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>90</y>
        <width>71</width>
        <height>281</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab3</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
